examples/vision/md/object_detection_using_vision_transformer.md

```diff
@@ -388,7 +386,7 @@ input_shape = (image_size, image_size, 3)  # input image shape
 learning_rate = 0.001
 weight_decay = 0.0001
 batch_size = 32
-num_epochs = 100
+num_epochs = 15
```

Comment:
Add a comment specifying that the real number of epochs should be 100.

Reply:
Set back to 100, since it ran fast enough.